Example/Example/ComicsNavigationController.swift

```diff
@@ -8,12 +8,16 @@
 
 import Foundation
 import UIKit
+import BothamUI
 
 class ComicsNavigationController: UINavigationController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        let viewController = AppDelegate.service.provideComicsViewController()
-        pushViewController(viewController, animated: false)
+
+        let navigator = ServiceLocator.SharedInstance.navigatorContainer.register("Comics", navigationController: self)
```

Comment:
I would try to not use the ServiceLocator from the UseCases to the left (presenter, VC)